<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="eventLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>721</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="mplfigure" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1001</width>
      <height>384</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>512</width>
      <height>384</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="pulse">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pulse_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuECG_Monitor">
    <property name="title">
     <string>ECG Monitor</string>
    </property>
    <addaction name="actionBluetooth_Starte"/>
    <addaction name="actionDaten_Transfer_starten"/>
    <addaction name="actionPause"/>
    <addaction name="actionDaten_Aufzeichnen"/>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuECG_Monitor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBluetooth_Starte">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bluetooth Starten</string>
   </property>
  </action>
  <action name="actionDaten_Transfer_starten">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Daten Transfer starten</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionDaten_Aufzeichnen">
   <property name="text">
    <string>Daten Aufzeichnen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
